src/Mailer/Mailer.php

```diff
@@ -212,7 +212,7 @@ public function viewBuilder()
      * @param array $args Method arguments
      * @return $this|mixed
      */
-    public function __call($method, $args)
+    public function __call(string $method, array $args)
```

Comment:
Adding typehints to magic methods seems quite unnecessary as these methods are called by PHP itself not the user.

Reply:
OK